### DISCUSSION TITLE
LUKS with btrfs?

### DISCUSSION THREAD

**Discussion by Framsfex** [linuxmint/discussions#198] [Ideas]
I tried to install Mint with LUKS and btrfs for /, but I failed.
With the Mint Installer I can only install LUKS together with ext4, there is no option for btrfs.
For several reasons I need btrfs for the / filesystem AND full encryption (LUKS).
I was using linuxmint-21.2-xfce-64bit.iso

**Comment by eco-star** [comment]
I would appreciate **btrfs** in the installer aswell. Now that timeshift is part of the mint-apps, it would make sense to add the possibility directly to the installer.

**Comment by zelimir86** [comment]
I think it is must have feature. A lot of distros give this option during install (Fedora, Arch based distros with Calamares installer)
